src/pages/code-of-conduct.js

```diff
@@ -0,0 +1,172 @@
+import React from "react";
+import Layout from "../components/layout/layout";
+const CodeOfConduct = () => {
+  return (
+    <Layout>
+      <h2 className="text-center mt-4">Code of conduct</h2>
+      <img className="mx-auto my-5" src="/static/codeconduct.svg" />
```

Comment:
La imagen codeconduct al parecer no está en el proyecto, por lo que no se está mostrando en la página

![image](https://user-images.githubusercontent.com/20668624/96213203-0a4adf00-0f3e-11eb-931a-81d531e6cf8c.png)

Reply:
Podrías exportarla desde el prototipo:

![image](https://user-images.githubusercontent.com/20668624/96213566-f2c02600-0f3e-11eb-8bfe-1012c8c28101.png)

Reply:
La exporte incluso la puse en static como svg

Reply:
```suggestion
      <img className="mx-auto my-5" src={codeConductImg} />
```
Si haces la importación como te la pongo arriba aqui simplemente la usas asi